test/specs/templates.e2e.ts

```diff
@@ -27,6 +27,220 @@ describe('Templates', async () => {
     projectName = testSetup.tempProjectName.toUpperCase();
   });
 
+  // Apex Class
+  step('Create an Apex Class', async () => {
+    // Using the Command palette, run SFDX: Create Apex Class.
+    const workbench = await browser.getWorkbench();
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+    // Set the name of the new component to ApexClass1.
+    await inputBox.setText('ApexClass1');
+    await inputBox.confirm();
+    await utilities.pause(1);
+
+    // Select the default directory (press Enter/Return).
+    await inputBox.confirm();
+
+    const successNotificationWasFound = await utilities.attemptToFindNotification(workbench, 'SFDX: Create Apex Class successfully ran', 10);
+    expect(successNotificationWasFound).toBe(true);
+
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'Finished SFDX: Create Apex Class', 10);
+    expect(outputPanelText).not.toBeUndefined();
+
+    // Check for expected items in the Explorer view.
+    const sidebar = workbench.getSideBar();
+    const treeViewSection = await sidebar.getContent().getSection(projectName);
+    await treeViewSection.expand();
+
+    // Get the matching (visible) items within the tree which contains "apexClass1".
+    const filteredTreeViewItems = await utilities.getFilteredVisibleTreeViewItemLabels(workbench, projectName, 'ApexClass1');
+
+    expect(filteredTreeViewItems.includes('ApexClass1.cls')).toBe(true);
+    expect(filteredTreeViewItems.includes('ApexClass1.cls-meta.xml')).toBe(true);
+  });
+
+  step('Verify the contents of the Apex Class', async () => {
+    const expectedText = [
+      'public with sharing class ApexClass1 {',
+      '    public ApexClass1() {',
+      '',
+      '    }',
+      '}'
+    ].join('\n');
+    const workbench = await browser.getWorkbench();
+    const editorView = await workbench.getEditorView();
+    const textEditor = await editorView.openEditor('ApexClass1.cls') as TextEditor;
```

Comment:
Sometimes this doesn't work on windows when the Scratch Org is reused because it doesn't create the class again hence it doesn't automatically open the editor, and this works for opened editors. I added a couple of lines in [my PR](https://github.com/forcedotcom/salesforcedx-vscode-automation-tests/pull/17/files#diff-793780c191ff05e8020a677ee69fc22fd1399a2531a4c3bb8b64010b1a9d8b00R27) as a workaround

Reply:
OK, I'll merge in what I have her, and Windows-specific issues can be addressed in your PR.